#ifndef WEB_SERVICE_H
#define WEB_SERVICE_H

#include <algorithm>
#include <chrono>
#include <iostream>
#include <memory>
#include <random>
#include <stdio.h>
#include <string>
#include <thread>
#include <unistd.h>

#include <grpc/grpc.h>
#include <grpcpp/channel.h>
#include <grpcpp/client_context.h>
#include <grpcpp/create_channel.h>
#include <grpcpp/security/credentials.h>

//includi zbuildane kode -> TODO popravi pot
#include "../storage.grpc.pb.h"

#endif
